radicle-cli/src/commands/tag.rs

```diff
@@ -49,7 +50,7 @@ impl Args for Options {
                 Value(ref val) if id.is_none() => {
                     let val = val.to_string_lossy();
                     let Ok(val) = cob::ObjectId::from_str(&val) else {
-                        return Err(anyhow!("invalid issue or patch ID '{}'", val));
+                        return Err(anyhow!("invalid id for issue or patch '{}'", val));
```

Comment:
Can we use one of the special functions here?

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-8f93a6b75941543d5dca23ad1f7edccf33cd863f5149b83b899d48d91c73560eR51